packages/interactivity/README.md

```diff
@@ -0,0 +1 @@
+# Interactivity
```

Comment:
@juanmaguitar can you help us with this?

Reply:
Is this published in the handbook somehow? We don't want that until this package is not experimental anymore. Or if there's no way to avoid that, it should only say that it's experimental and where to find more information to get started.

Reply:
Ok, let's merge this with an empty Readme and do it in another PR tomorrow.